assets/css/default.css

```diff
@@ -5,6 +5,17 @@ body {
     Arial, sans-serif;
 }
 
+.length-watch-page{
+  position: relative;
+  background-color: rgba(35, 35, 35, 0.75);
+  color: #fff;
+  border-radius: 2px;
+  padding: 4px;
```

Comment:
This could see some left/right margins! The box looks too close to the text.

Reply:
By text you mean the video title or the text inside the box?

Reply:
I meant the video title